src/sdk/bucketspace_fs.js

```diff
@@ -156,7 +156,8 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             };
 
             bucket.name = new SensitiveString(bucket.name);
-            bucket.system_owner = new SensitiveString(bucket.system_owner);
+            // we want to remove the system_owner, but we might have buckets that were created in the past that had this property
+            if (bucket.system_owner) bucket.system_owner = new SensitiveString(bucket.system_owner);
```

Comment:
I think this code is not needed now. 
We can just ignore/remove it from the object when loading bucket info from json.

Reply:
I need it until I will implement the read-modify-write